templates/win_userdata.ps1

```diff
@@ -254,17 +269,15 @@ function Install-PythonGit {
   $GitUrl = "${url_git}"
 
   # Download bootstrap file
-  $Stage = "download bootstrap"
   $BootstrapFile = "$${Env:Temp}\$($${BootstrapUrl}.split("/")[-1])"
   (New-Object System.Net.WebClient).DownloadFile($BootstrapUrl, $BootstrapFile)
 
-  # Install python and git
-  $Stage = "install python/git"
   & "$BootstrapFile" `
-      -PythonUrl "$PythonUrl" `
-      -GitUrl "$GitUrl" `
-      -Verbose -ErrorAction Stop
-  Test-DisplayResult "Install Python/Git" $?
+    -PythonUrl "$PythonUrl" `
+    -GitUrl "$GitUrl" `
+    -Verbose -ErrorAction Stop
+  $Script = "$BootstrapFile -PythonUrl $PythonUrl -GitUrl $GitUrl -Verbose -ErrorAction Stop"    
+  Test-DisplayResult "Install Python/Git [$Script]" $?
```

Comment:
won't `$?` here capture the result of the `$Script` assignment on the prior line, not the return code from the bootstrap script?

Reply:
Yes. I fixed it. I've tried several things to make that line work with one string that gets executed and logged but none have worked. That was a missed artifact of those attempts.